lib/src/animated_map_controller.dart

```diff
@@ -145,6 +148,7 @@ class AnimatedMapController {
       vsync: vsync,
       duration: duration ?? this.duration,
     );
+    stopAndDisposeAnimationController();
```

Comment:
I'm not a big fan of this, the idea is not to stop a previous animation when another one is triggered.
Maybe a better way could be to have a private `Set` of the running animation controllers, so when the `dispose` method is called it could iterate over the remaining controllers to dispose of them. 🤔